django/thunderstore/social/api/experimental/views/complete_login.py

```diff
@@ -0,0 +1,67 @@
+from django.conf import settings
+from django.http import HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import serializers
+from rest_framework.response import Response
+from rest_framework.status import HTTP_400_BAD_REQUEST
+from rest_framework.views import APIView
+
+from thunderstore.social.providers import DiscordOauthHelper, GitHubOauthHelper
+
+
+class RequestBody(serializers.Serializer):
+    code = serializers.CharField(
+        label="Temporary code received from provider when authentication flow was initiated on client"
```

Comment:
The code has a specific term in the OAuth spec which would make sense to include in here for docs, though I don't remember what was it called ([authorization code perhaps?](https://datatracker.ietf.org/doc/html/rfc6749#section-1.3.1)).

Reply:
Changed the term.